packages/amplify-util-node-template-provider/src/utils/analyticsWalkthrough.ts

```diff
@@ -0,0 +1,64 @@
+import inquirer from 'inquirer'
+
+export async function askAnalyticsCategoryKinesisQuestions(context: any) {
+  const { amplify } = context;
+  const { allResources } = await amplify.getResourceStatus();
+  const kinesisResources = allResources.filter((resource: any) => resource.service === 'Kinesis');
+
+  let targetResourceName;
+  if (kinesisResources.length === 0) {
+    context.print.error('No Kinesis streams resource to select. Please use "amplify add analytics" command to create a new Kinesis stream');
+    process.exit(0);
+    return;
```

Comment:
do we need this here?

Reply:
This is all copied from ambientlight's PR. I'm not too keen to touch it too much.